### ISSUE TITLE
State the file descriptor lifetime passed with VASurfaceAttribExternalBuffers

### ISSUE THREAD

**Issue by rosetta-jpn** [intel/libva#551]
It is unclear that the lifetime of file descriptor passed with
VASurfaceAttribExternalBuffers(https://github.com/intel/libva/blob/6e86b4fb4dafa123b1e31821f61da88f10cfbe91/va/va.h#L1664) and VA_SURFACE_ATTRIB_MEM_TYPE_DRM_PRIME.
As far as I looked media-driver implementation, the created VASurface by CreateSurfaces2() with the attribute creates and owns GEM object created from the passed fd.
It is safe to close the passed fd.
However, it is not guaranteed in VA API level.

**Comment by XinfengZhang** [comment]
yes, after the fd is imported into  driver, the file descriptor is safe to be closed.

**Comment by rosetta-jpn** [comment]
Thanks for clarifying.
May I ask to guarantee it in VA-API level by explicitly documenting in the code?

**Comment by XinfengZhang** [comment]
good idea, I filed #553 to handle it, please help to review it,
